package.json

```diff
@@ -37,15 +37,15 @@
     "chalk": "^4.1.2",
     "clang-format": "^1.8.0",
     "dmd-clear": "^0.1.2",
-    "eslint": "^8.39.0",
-    "eslint-config-prettier": "^8.8.0",
+    "eslint": "^8.46.0",
+    "eslint-config-prettier": "^8.10.0",
     "eslint-plugin-prettier": "^4.2.1",
     "jsdoc-to-markdown": "^8.0.0",
     "mocha": "^10.2.0",
-    "mongodb": "^5.3.0",
+    "mongodb": "^5.7.0",
```

Comment:
Is there any reason to not update the mongodb version as well?

Reply:
Fine with me to update it.